### PR TITLE
Tutorial updates

### LINKED ISSUE #1563: [Bug] Invalid Solver Type for Tutorial 6 (CO2 injection)
The solver type provided in the xml for tutorial 6 is not supported by Trilinos. The xml sets the solver type to "fgmres" but this solver type is not supported in the Trilinos interface code. 

**To Reproduce**
1. run: mpirun -np 4 ./bin/geosx -i ../src/coreComponents/physicsSolvers/multiphysics/integratedTests/FieldCaseCo2InjTutorial.xml
2. See error
```
***** ERROR
***** LOCATION: /g/g15/neth3/climate/GEOSX/src/coreComponents/linearAlgebra/interfaces/trilinos/TrilinosSolver.cpp:61
***** Controlling expression (should be false): true
***** Rank 1: Solver type not supported in Trilinos interface: fgmres

** StackTrace of 13 frames **
Frame 0: geosx::TrilinosSolver::TrilinosSolver(geosx::LinearSolverParameters)
Frame 1: std::_MakeUniq<geosx::TrilinosSolver>::__single_object std::make_unique<geosx::TrilinosSolver, geosx::LinearSolverParameters>(geosx::LinearSolverParameters&&)
Frame 2: geosx::TrilinosInterface::createSolver(geosx::LinearSolverParameters)
Frame 3: geosx::SolverBase::solveSystem(geosx::DofManager const&, geosx::EpetraMatrix&, geosx::EpetraVector&, geosx::EpetraVector&)
Frame 4: geosx::ReservoirSolverBase::solveSystem(geosx::DofManager const&, geosx::EpetraMatrix&, geosx::EpetraVector&, geosx::EpetraVector&)
Frame 5: geosx::SolverBase::nonlinearImplicitStep(double const&, double const&, int, geosx::DomainPartition&)
Frame 6: geosx::ReservoirSolverBase::solverStep(double const&, double const&, int, geosx::DomainPartition&)
Frame 7: geosx::SolverBase::execute(double, double, int, int, double, geosx::DomainPartition&)
Frame 8: geosx::EventBase::execute(double, double, int, int, double, geosx::DomainPartition&)
Frame 9: geosx::EventManager::run(geosx::DomainPartition&)
Frame 10: geosx::ProblemManager::runSimulation()
Frame 11: geosx::GeosxState::run()
Frame 12: main
Frame 13: /lib64/libc.so.6
=====
```

**Platform (please complete the following information):**
- Machine: LLNL/Lassen
- Compiler: clang upstream 2019.03.26 
- GEOSX Version: develop branch

**Additional context**
I went into the xml file and changed it from fgmres to gmres and the above error went away. However, I get a new, similar error:
```
***** ERROR
***** LOCATION: /g/g15/neth3/climate/GEOSX/src/coreComponents/linearAlgebra/interfaces/trilinos/TrilinosPreconditioner.cpp:304
***** Controlling expression (should be false): true
***** Rank 1: Preconditioner type not supported in Trilinos interface: mgr

** StackTrace of 11 frames **
Frame 0: geosx::TrilinosSolver::setup(geosx::EpetraMatrix const&)
Frame 1: geosx::SolverBase::solveSystem(geosx::DofManager const&, geosx::EpetraMatrix&, geosx::EpetraVector&, geosx::EpetraVector&)
Frame 2: geosx::ReservoirSolverBase::solveSystem(geosx::DofManager const&, geosx::EpetraMatrix&, geosx::EpetraVector&, geosx::EpetraVector&)
Frame 3: geosx::SolverBase::nonlinearImplicitStep(double const&, double const&, int, geosx::DomainPartition&)
Frame 4: geosx::ReservoirSolverBase::solverStep(double const&, double const&, int, geosx::DomainPartition&)
Frame 5: geosx::SolverBase::execute(double, double, int, int, double, geosx::DomainPartition&)
Frame 6: geosx::EventBase::execute(double, double, int, int, double, geosx::DomainPartition&)
Frame 7: geosx::EventManager::run(geosx::DomainPartition&)
Frame 8: geosx::ProblemManager::runSimulation()
Frame 9: geosx::GeosxState::run()
Frame 10: main
Frame 11: /lib64/libc.so.6
=====
```

Changing it to amg within the xml fixes this error as well.

### PR DESCRIPTION
**Pull request by joshua-white**
Here is the proposed restructuring of the Tutorials and Examples section, based on our doc day discussion.  Note that some of this material doesn't exist yet, but we are just noting where it could go in future PRs.

Also, as a general note, let's discourage three-level nesting of headers where possible. Two provides more info to the reader in the sidebar.

Tutorials 
--------
1. First Steps
	- improve visualizations (VTK)
2. External Meshes
3. Regions and Property Specification
4. Boundary  Conditions and Time-Dependent Functions
	- Move elastic beam tutorial here
5. Time-History Collection 

Basic Examples
---------------
1. Multiphase Flow 
2. Multiphase Flow with Wells 
3. CO2 Injection
	- replace with simple plume model, faster, larger CO2 plume
4. Poromechanics (replace Terzaghi with Mandel)
5. Hydraulic Fracturing
6. Triaxial Driver
7. pygeosx

Advanced Examples
--------------------
1. Validation Studies
	- Buckley-Leverett
	- Sedov
	- Kirsch Solution
	- Poroelastic Wellbore
	- Poroplastic Wellbore
	- Terzaghi's & Mandel's Problem
	- Sneddon's Problem
	- Single-Fracture Compression
	- PKN, KGD, Penny Shape
	- Laplace (VEM, FEM)
	- Pruess CO2 validation examples
	- Deviated wellbores and wellbores with cement
1. Performance Benchmarks
	- SPE10++ Flow
	- SPE10++ Poromechanics
	- Sedov
	- Staircase Benchmark
	- MGR examples: hydraulic fracturing, mimetic, compositions with wells
	- ECP results with wellbore mesh
	- Sergey Multiscale
	- Jacques Indicators
2. Application Studies
	- High Island 24L: Single-Phase Poromechanics
	- High Island 24L: Compositional Poromechanics
	- EERE Model (Hewei's Case)
	- Fault Mechanics (Andrea)
	- Phase Field (Andre)
	- Acoustic & Elastic Wave Propagation
	
	
	Also: fixes #1563

### CODE REVIEW

**Comment by francoishamon** [comment]
The new structure of the Tutorials/Examples is in place and can be looked at here: https://geosx-geosx.readthedocs-hosted.com/en/docs-tutorialupdates/index.html

File: src/docs/sphinx/basicExamples/multiphaseFlowWithWells/Example.rst [hunk 1]
```diff
@@ -255,21 +261,22 @@ of the Egg test case.
 
 .. _FieldSpecifications_tag_dead_oil_egg_model:
 
-Defining properties with the FieldSpecifications
----------------------------------------------------------------------
+-----------------------
+Initial condition
```
Comment (sytuannguyen) [inline src/docs/sphinx/basicExamples/multiphaseFlowWithWells/Example.rst:265]:
```suggestion
Initial conditions
```

File: src/docs/sphinx/basicExamples/poromechanics/Example.rst [hunk 2]
```diff
@@ -171,7 +167,7 @@ The parameters used in the simulation are summarized in the following table.
 +================+=======================+==================+===================+
 | :math:`E`      | Young's modulus       | [Pa]             | 1.0*10\ :sup:`4`  |
```
Comment (sytuannguyen) [inline src/docs/sphinx/basicExamples/poromechanics/Example.rst:168]:
```suggestion
| :math:`E`      | Young's modulus       | [Pa]             | 10\ :sup:`4`  |
```

**Review by sytuannguyen — COMMENTED** [review]
Just some typo corrections

File: src/coreComponents/fileIO/doc/InputXMLFiles.rst [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+xmllint is a command-line tool that is typically pre-installed on unix-like systems.
+To check whether an input file is valid, run the following command:
+
+xmllint --schema /path/to/schema.xsd input_file.xml
+
+
+Sublime Text
+------------------------------
+
+We reccomend using the `Exalt <https://github.com/eerohele/exalt>`_ or `SublimeLinter_xmllint <https://github.com/SublimeLinter/SublimeLinter-xmllint>`_ plug-ins to validate xml files within sublime.
```
[the remark this answers is not in the document]
Comment (francoishamon) [reply src/coreComponents/fileIO/doc/InputXMLFiles.rst]: Done

File: src/coreComponents/fileIO/doc/InputXMLFiles.rst [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+            },
+        }
+    }
+
+
+
+Eclipse
+------------------------------
+
+To Eclipse Web Develop Tools includes features for validating xml files.
```
[the remark this answers is not in the document]
Comment (francoishamon) [reply src/coreComponents/fileIO/doc/InputXMLFiles.rst]: Done

File: src/coreComponents/fileIO/doc/InputXMLFiles.rst [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+To install them, go to Help -> Eclipse Marketplace, search for the Eclipse Web Developer Tools, install the package, and restart eclipse.
```
[the remark this answers is not in the document]
Comment (francoishamon) [reply src/coreComponents/fileIO/doc/InputXMLFiles.rst]: Done

File: src/coreComponents/fileIO/doc/InputXMLFiles.rst [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+Eclipse will automatically fetch the schema, and validate an active xml file.
+The editor will highlight any lines with errors, and underline the specific errors.
+
+
+GEOSX XML Tools
+------------------------------
+
+The geosx_xml_tools package, which is used to enable advanced features such as parameters, symbolic math, etc., contains tools for validating xml files.
+To do so, call the command-line script with the -s argument, i.e.: `preprocess_xml input_file.xml -s /path/to/schema.xsd`.
+After compiling the final xml file, pygeos will fetch the designated schema, validate, and print any errors to the screen.
```
[the remark this answers is not in the document]
Comment (francoishamon) [reply src/coreComponents/fileIO/doc/InputXMLFiles.rst]: Done

File: src/coreComponents/fileIO/doc/InputXMLFiles.rst [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+An XML schema definition (XSD) file lays out the expected structure of an input XML file.
+During the build process, GEOSX automatically constructs a comprehensive schema from the code's data structure, and updates the version in the source (GEOSX/src/coreComponents/schema/schema.xsd).
+
+
+Schema Components
+------------------------------
+
+The first entry in the schema are a set of headers the file type and version.
+Following this, the set of available simple types for attributes are layed out.
+Each of these include a variable type name, which mirrors those used in the main code, and a regular expression, which is designed to match valid inputs.
```
[the remark this answers is not in the document]
Comment (francoishamon) [reply src/coreComponents/fileIO/doc/InputXMLFiles.rst]: Done

File: src/docs/sphinx/basicExamples/hydraulicFracturing/Example.rst [hunk 8]
```diff
@@ -151,14 +147,14 @@ Therefore, it is important to test whether the size of the model is large enough
...
   :end-before: <!-- SPHINX_HYDROFRACTURE_BC_BASE_END -->
 
-
-Solvers: setting up the coupled hydraulic fracturing solver
+-----------------------------------------------------------------
+Coupled hydraulic fracturing solver
 -----------------------------------------------------------------
 
 The Solvers block is located in the base xml file.
 Note that the ``gravityVector`` attribute indicates that we are applying gravity in the -z direction in this problem.
```
[the remark this answers is not in the document]
Comment (francoishamon) [reply src/docs/sphinx/basicExamples/hydraulicFracturing/Example.rst]: Done

File: src/docs/sphinx/basicExamples/hydraulicFracturing/Example.rst [hunk 9]
```diff
@@ -106,9 +101,9 @@ The other dimensions of the mesh follow a similar pattern.
...
   :end-before: <!-- SPHINX_HYDROFRACTURE_MESH_END -->
 
-
-Geometry: defining a fracture nodeset
---------------------------------------------------------------
+----------------------------------------------
+Defining a fracture nodeset
+----------------------------------------------
 
 For this example, we want to propagate a single hydraulic fracture along the y=0 plane.
```
[the remark this answers is not in the document]
Comment (francoishamon) [reply src/docs/sphinx/basicExamples/hydraulicFracturing/Example.rst]: Done

File: src/docs/sphinx/basicExamples/hydraulicFracturing/Example.rst [hunk 10]
```diff
@@ -42,19 +42,14 @@ The in-situ properties and initial conditions are based upon a randomly generate
...
-
-------------------------------------------------------------------
-Preparing the input files
-------------------------------------------------------------------
-
 The inputs for this case are contained inside a case-specific (``heterogeneousInSitu_singleFracture.xml``) and base (``heterogeneousInSitu_base.xml``) XML file.
 The ``tables`` directory contains the pre-constructed geologic model.
-This tutorial will first focus on the case-specific input file, which contains the key parameter definitions, then consider the base-file.
-
+This example will first focus on the case-specific input file, which contains the key parameter definitions, then consider the base-file.
```
[the remark this answers is not in the document]
Comment (francoishamon) [reply src/docs/sphinx/basicExamples/hydraulicFracturing/Example.rst]: Done

File: src/docs/sphinx/basicExamples/poromechanics/Example.rst (hunk 2 above)
Comment (francoishamon) [reply src/docs/sphinx/basicExamples/poromechanics/Example.rst]: Done

File: src/docs/sphinx/basicExamples/multiphaseFlowWithWells/Example.rst (hunk 1 above)
Comment (francoishamon) [reply src/docs/sphinx/basicExamples/multiphaseFlowWithWells/Example.rst]: Done

File: src/coreComponents/fileIO/doc/InputXMLFiles.rst [hunk 11]
```diff
@@ -0,0 +1,173 @@
...
+These patterns are defined and documented in ``DataTypes::typeRegex``.
+The final part of the schema is the file layout, beginning with the root ``Problem``.
+Each complex type defines an element, its children, and its attributes.
+Each attribute defines the input name, type, default value, and/or usage.
+Comments preceding each attribute are used to relay additional information to the user.
```
[the remark this answers is not in the document]
Comment (francoishamon) [reply src/coreComponents/fileIO/doc/InputXMLFiles.rst]: Done

File: src/docs/sphinx/advancedExamples/validationStudies/sneddon/Example.rst [hunk 12]
```diff
@@ -113,36 +113,35 @@ The parameters used in the simulation are summarized in the following table.
   +================+=======================+==================+===================+
   | :math:`E`      | Young's modulus       | [Pa]             | 1.0*10\ :sup:`4`  |
```
[the remark this answers is not in the document]
Comment (francoishamon) [reply src/docs/sphinx/advancedExamples/validationStudies/sneddon/Example.rst]: Done

File: src/docs/sphinx/advancedExamples/validationStudies/sneddon/Example.rst [hunk 13]
```diff
@@ -113,36 +113,35 @@ The parameters used in the simulation are summarized in the following table.
...
   +----------------+-----------------------+------------------+-------------------+
-  | :math:`\nu`    | Poisson's ration      | [-]              | 0.2               |
+  | :math:`\nu`    | Poisson's ratio       | [-]              | 0.2               |
   +----------------+-----------------------+------------------+-------------------+
   | :math:`L_f`    | Fracture length       | [m]              | 20                |
   +----------------+-----------------------+------------------+-------------------+
-  | :math:`p_f`    | fracture pressure     | [Pa]             | 1.0*10\ :sup:`5`  |
+  | :math:`p_f`    | Fracture pressure     | [Pa]             | 1.0*10\ :sup:`5`  |
```
[the remark this answers is not in the document]
Comment (francoishamon) [reply src/docs/sphinx/advancedExamples/validationStudies/sneddon/Example.rst]: Done

File: src/docs/sphinx/basicExamples/co2Injection/Example.rst [hunk 14]
```diff
@@ -0,0 +1,322 @@
...
+   
+
+.. _Mesh_tag_co2_field_case:
+
+
+----------------------
+Mesh and well geometry
+----------------------
+
+In this example, the **Mesh** tag is used to generate the reservoir mesh internally (:ref:`TutorialSinglePhaseFlowWithInternalMesh`). The internal generation of well is defined with the **InternalWell** sub-tag. Apart from the ``name`` identifier attribute and their ``wellRegionName`` (:ref:`ElementRegions <ElementRegions_tag_co2_field_case>`) and ``wellControlsName`` (:ref:`Solver <Solver_tag_co2_field_case>`) binding attributes, ``polylineNodeCoords`` and ``polylineSegmentConn`` attributes are used to define the path of the wellbore and connections between its nodes. The ``numElementsPerSegment`` is discretizing the wellbore segments while the ``radius`` attribute specifies the wellbore radius (:ref:`TutorialDeadOilEgg` for details on wells). Once the wellbore is defined and discretized, the position of **Perforations** is defined using the linear distance from the head of the wellbore (``distanceFromHead``).
```
[the remark this answers is not in the document]
Comment (francoishamon) [reply src/docs/sphinx/basicExamples/co2Injection/Example.rst]: Done

File: src/docs/sphinx/basicExamples/co2Injection/Example.rst [hunk 15]
```diff
@@ -0,0 +1,322 @@
...
+.. _Events_tag_co2_field_case:
+
+--------
+Events
+--------
+        
+The solver is applied as a periodic event whose target is referred to as ``coupledFlowAndWells`` nametag.
+Using the ``maxEventDt`` attribute, we specify a max time step size of 5 x 10^6 seconds.
+
+The output event triggers a VTK output  every 10^7 seconds, constraining the solver schedule to match exactly these dates.
```
[the remark this answers is not in the document]
Comment (francoishamon) [reply src/docs/sphinx/basicExamples/co2Injection/Example.rst]: Done

File: src/docs/sphinx/basicExamples/co2Injection/Example.rst [hunk 16]
```diff
@@ -0,0 +1,322 @@
...
+The output path to data is specified as a ``target`` of this **PeriodicEvent**.
+
+Another periodic event is defined under the name ``restarts``.
+It consists of saved checkpoints every 5 x 10^7 seconds, whose physical output folder name is defined under the **Output** tag.
```
[the remark this answers is not in the document]
Comment (francoishamon) [reply src/docs/sphinx/basicExamples/co2Injection/Example.rst]: Done

File: src/docs/sphinx/basicExamples/co2Injection/Example.rst [hunk 17]
```diff
@@ -0,0 +1,322 @@
...
+  :end-before: <!-- SPHINX_FIELD_CASE_Co2_NUMERICAL_END -->
+
+.. _ElementRegions_tag_co2_field_case:
+
+---------------
+Element regions
+---------------
+
+We define a **CellElementRegion** pointing to the cell block defining the reservoir mesh, and a **WellElementRegion** for the well.
+The two regions contain a list of constitutive models names.
```
[the remark this answers is not in the document]
Comment (francoishamon) [reply src/docs/sphinx/basicExamples/co2Injection/Example.rst]: Done

File: src/docs/sphinx/basicExamples/co2Injection/Example.rst [hunk 18]
```diff
@@ -0,0 +1,322 @@
...
+
+-----------------
+Constitutive laws
+-----------------
+
+Under the **Constitutive** tag, four items can be found:
+
+- **CO2BrineFluid** : this tag defines phase names, component molar weights, and fluid behaviors such as CO :sub:`2` solubility in brine and viscosity/density dependencies on pressure and temperature. 
+- **PressurePorosity** : this tag contains all the data needed to model rock compressibility.
+- **BrooksCoreyRelativePermeability** : this tag defines the relative permeability model for each phase, its end-point values, residual volume fractions (saturations), and Corey exponents.
```
[the remark this answers is not in the document]
Comment (francoishamon) [reply src/docs/sphinx/basicExamples/co2Injection/Example.rst]: Done

File: src/docs/sphinx/basicExamples/co2Injection/Example.rst [hunk 19]
```diff
@@ -0,0 +1,322 @@
...
+The model is described in detail in  :ref:`CO2-EOS`.
+The model definition requires three text files:
+
+In *co2flash.txt*, we define the CO :sub:`2` solubility model used to compute the amount of CO :sub:`2` dissolved in the brine phase as a function of pressure (in Pascal), temperature (in Kelvin), and salinity (in units of molality):
+
+.. code::
+
+   FlashModel CO2Solubility  1e6 1.5e7 5e4 367.15 369.15 1 0
+
+The first keyword is an identifier for the model type (here, a flash model). It is followed by the model name. Then, the lower, upper, and step increment values for pressure and temperature range are specified.
```
[the remark this answers is not in the document]
Comment (francoishamon) [reply src/docs/sphinx/basicExamples/co2Injection/Example.rst]: Done

File: src/docs/sphinx/basicExamples/co2Injection/Example.rst [hunk 20]
```diff
@@ -0,0 +1,322 @@
...
+        ViscosityFun FenghourCO2Viscosity 1e6 1.5e7 5e4 94 96 
+
+.. code::
+  
+        DensityFun BrineCO2Density 1e6 1.5e7 5e4 94 96 1 0
+        ViscosityFun BrineViscosity 0
+
+In these files, the first keyword of each line is an identifier for the model type (either a density or a viscosity model).
+It is followed by the model name.
+Then, the lower, upper, and step increment values for pressure and temperature range are specified.
```
[the remark this answers is not in the document]
Comment (francoishamon) [reply src/docs/sphinx/basicExamples/co2Injection/Example.rst]: Done

File: src/docs/sphinx/basicExamples/co2Injection/Example.rst [hunk 21]
```diff
@@ -0,0 +1,322 @@
...
+The trailing 0 for ``BrineCO2Density`` and ``BrineViscosity`` entry is the salinity of the brine, set to zero.
+
+.. note::
+   It is the responsibility of the user to make sure that the pressure and temperature values encountered in the simulation (in the reservoir and in the well) are within the bounds specified in the PVT files. GEOSX will not throw an error if a value outside these bounds is encountered, but the (nonlinear) behavior of the simulation and quality of the results will likely be negatively impacted.
```
[the remark this answers is not in the document]
Comment (francoishamon) [reply src/docs/sphinx/basicExamples/co2Injection/Example.rst]: Done

File: src/docs/sphinx/basicExamples/co2Injection/Example.rst [hunk 22]
```diff
@@ -0,0 +1,322 @@
...
+
+------
+Output
+------
+
+The **Outputs** XML tag is used to write visualization, restart, and time history files.
+
+Here, we write visualization files in a format natively readable by Paraview under the tag **VTK**.
+A **Restart** tag is also be specified. In conjunction with a **PeriodicEvent**,
+a restart file allows to resume computations from a set checkpoint in time.
```
[the remark this answers is not in the document]
Comment (francoishamon) [reply src/docs/sphinx/basicExamples/co2Injection/Example.rst]: Done

File: src/docs/sphinx/basicExamples/co2Injection/Example.rst [hunk 23]
```diff
@@ -0,0 +1,322 @@
...
+.. literalinclude:: ../../../../coreComponents/physicsSolvers/multiphysics/integratedTests/SimpleCo2InjTutorial.xml
+  :language: xml
+  :start-after: <!-- SPHINX_FIELD_CASE_Co2_TASKS -->
+  :end-before: <!-- SPHINX_FIELD_CASE_Co2_TASKS_END -->
+
+.. 
+        ------------------------------------------------
+        Using Functions to specify dependent properties
+        ------------------------------------------------
+        The *TableFunctions* are used to map heterogeneous properties (here permeabiity and porosity) onto the mesh via the reading of regular grid coordinate in *xlin.geos*, *ylin.geos* and *zlin.geos* and a ND-table at *voxelFile* adress containing heterogenous data. (see :ref:`TutorialFieldCase` for more details on those)
```
[the remark this answers is not in the document]
Comment (francoishamon) [reply src/docs/sphinx/basicExamples/co2Injection/Example.rst]: Removed this part (was commented out)

File: src/docs/sphinx/basicExamples/co2Injection/Example.rst [hunk 24]
```diff
@@ -0,0 +1,322 @@
...
+        Rank 3: Reading in restart file at SimpleCo2InjTutorial_restart_000000024/rank_0000003.hdf5
+        Rank 2: Reading in restart file at SimpleCo2InjTutorial_restart_000000024/rank_0000002.hdf5
+
+and the simulation restarts from this point in time. 	
+
+-------------
+Visualization
+-------------
+
+Post-treating in Paraview, we can observe the CO :sub:`2` plume moving upward under buoyancy effects and
```
[the remark this answers is not in the document]
Comment (francoishamon) [reply src/docs/sphinx/basicExamples/co2Injection/Example.rst]: Done

File: src/docs/sphinx/basicExamples/co2Injection/Example.rst [hunk 25]
```diff
@@ -0,0 +1,322 @@
...
+        Using Functions to specify dependent properties
+        ------------------------------------------------
+        The *TableFunctions* are used to map heterogeneous properties (here permeabiity and porosity) onto the mesh via the reading of regular grid coordinate in *xlin.geos*, *ylin.geos* and *zlin.geos* and a ND-table at *voxelFile* adress containing heterogenous data. (see :ref:`TutorialFieldCase` for more details on those) 
+
+
+------------------------------------
+Running GEOSX
+------------------------------------
+
+The simulation can be launched with on 4 cores using MPI-parallelism:
```
[the remark this answers is not in the document]
Comment (francoishamon) [reply src/docs/sphinx/basicExamples/co2Injection/Example.rst]: Done

File: src/docs/sphinx/basicExamples/hydraulicFracturing/Example.rst [hunk 26]
```diff
@@ -42,19 +42,14 @@ The in-situ properties and initial conditions are based upon a randomly generate
 .. image:: hf_example.png
 
 
-
-------------------------------------------------------------------
-Preparing the input files
-------------------------------------------------------------------
-
 The inputs for this case are contained inside a case-specific (``heterogeneousInSitu_singleFracture.xml``) and base (``heterogeneousInSitu_base.xml``) XML file.
```
[the remark this answers is not in the document]
Comment (francoishamon) [reply src/docs/sphinx/basicExamples/hydraulicFracturing/Example.rst]: Done

File: src/docs/sphinx/basicExamples/co2Injection/Example.rst [hunk 27]
```diff
@@ -0,0 +1,322 @@
...
+|pic1| |pic2| |pic3|
+
+.. |pic1| image:: fcCo2-sat-0.png
+   :width: 32%
+.. |pic2| image:: fcCo2-sat-1.png
+   :width: 32%
+.. |pic3| image:: fcCo2-sat-2.png
+   :width: 32%
+
+The heterogeneous values of the log permeability field can also be visualized in Paraview:
```
[the remark this answers is not in the document]
Comment (francoishamon) [reply src/docs/sphinx/basicExamples/co2Injection/Example.rst]: done

**Comment by francoishamon** [comment]
Thanks @sytuannguyen for going through the tutorials!

**Comment by francoishamon** [comment]
Here is the status at the end of the work on this PR (that I propose to merge now):

The items not done yet are marked as **TODO** below

Tutorials 
--------
1. Tutorial 1: First Steps (improved visualizations)
2. Tutorial 2: External Meshes (improved visualizations)
3. Tutorial 3: Regions and Property Specification (improved visualizations)
4. Tutorial 4: Boundary  Conditions and Time-Dependent Functions (improved visualizations)
5. **TODO** Tutorial 5: Time-History Collection

Basic Examples
---------------
1. Multiphase Flow (simpler, wells removed, no more linear solver failure)
2. Multiphase Flow with Wells (no more linear solver failure)
3. CO2 Injection (simpler)
4. Poromechanics ( **TODO**: replace Terzaghi with Mandel)
5. Hydraulic Fracturing
6. **TODO** Triaxial Driver
7. **TODO** pygeosx

Advanced Examples
--------------------

New structure has been implemented with the following subsections

1. Validation and Verification Studies
	- **TODO** Buckley-Leverett
	- **TODO** Sedov
	- **TODO** Kirsch Solution
	- **TODO** Terzaghi's & Mandel's Problem
	- Sneddon's Problem
	- Elasto-Plastic Near-Well Deformation
	- Cased Elastic Wellbore Problem
	- Poromechanical Wellbore Problem
	- **TODO** Single-Fracture Compression
	- **TODO** PKN, KGD, Penny Shape
	- **TODO** Laplace (VEM, FEM)
	- **TODO** Pruess CO2 validation examples
	- **TODO** Deviated wellbores and wellbores with cement
1. Performance Benchmarks
	- **TODO** SPE10++ Flow
	- **TODO** SPE10++ Poromechanics
	- **TODO** Sedov
	- **TODO** Staircase Benchmark
	- **TODO** MGR examples: hydraulic fracturing, mimetic, compositions with wells
	- **TODO** ECP results with wellbore mesh
	- **TODO** Sergey Multiscale
	- **TODO** Jacques Indicators
2. Application Studies
	- **TODO** High Island 24L: Single-Phase Poromechanics
	- **TODO** High Island 24L: Compositional Poromechanics
	- **TODO** EERE Model (Hewei's Case)
	- **TODO** Fault Mechanics (Andrea)
	- **TODO** Phase Field (Andre)
	- **TODO** Acoustic & Elastic Wave Propagation

**Review by francoishamon — APPROVED** [review]

**Review by sytuannguyen — APPROVED** [review]

**Review by jhuang2601 — APPROVED** [review]
Great job!

**Pull request merged**